.editorconfig

```diff
@@ -25,16 +25,22 @@ dotnet_naming_rule.private_members_with_underscore.severity = suggestion
 
 dotnet_naming_symbols.private_fields.applicable_kinds           = field
 dotnet_naming_symbols.private_fields.applicable_accessibilities = private
+dotnet_naming_symbols.private_fields.required_modifiers = abstract,async,readonly,static # all except const
```

Comment:
This breaks how field names are generated. With this change when creating new fields based on ctor params it will be `this.someProperty` instead of `_someProperty`